deploy/olm-catalog/keycloak-operator/15.0.1/manifests/keycloak-operator.v15.0.1.clusterserviceversion.yaml

```diff
@@ -35,7 +35,10 @@ metadata:
           "apiVersion": "keycloak.org/v1alpha1",
           "kind": "KeycloakRealm",
           "metadata": {
-            "name": "example-keycloakrealm"
+            "name": "example-keycloakrealm",
+            "labels": {
+              "app": "sso"
+            }
```

Comment:
@jonathanvila As we discussed privately, we'll do this as part of the 15.0.2 release.

Reply:
:+1: